### PR TITLE
[Technical Support] LPS-114721 Accessibility - Required Fields

### PR DESCRIPTION
**Pull request by rijema**

### CODE REVIEW

**Comment by rijema** [comment]
ci:test:sf

**Comment by rijema** [comment]
ci:test:relevant

**Comment by rijema** [comment]
ci:test:stable

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ColorPicker/ColorPicker.es.js [hunk 1]
```diff
@@ -69,7 +70,7 @@ const ClayColorPickerWithState = ({
 		<ClayColorPicker
 			aria-describedby={describedBy}
 			aria-invalid={invalid}
-			aria-label="color"
+			aria-label={label}
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ColorPicker/ColorPicker.es.js:73]: @rijema, have you tested it with different languages locally? I understand it works for field having options, but I'm little unsure about the other fields without options.

**Comment by rijema** [comment]
Thank you, Renato!

Could you elaborate on what would be ‘fields having options’? I assumed it would be, for example, Select, Radio, Checkbox, etc ... But I didn't quite understand what the difference of their label would be to the other fields. They always received the field title values ​​of the standard language and the translation will only be done if there is any modification in the form build after the available language is selected, but the value will be passed to the aria-label attribute equally.

6F67B988-5C26-466F-B4BC-C526518FB6B7

In the case of I change purposefully the value of the field to suit a specific language, the aria-label will receive the value correctly. Is it your question?

C564451E-3266-4EF7-9A45-04FA5E417C09

Thanks again,
Richard.

**Comment by natocesarrego** [comment]
You already understood correctly, @rijema. 🙂 And I just have one more thing to say regarding your pull and detailed explanation...

leonardo di caprio

**Comment by natocesarrego** [comment]
ci:forward

**Comment by rijema** [comment]
Thanks, @natocesarrego !! It really made my day. :D

**Comment by natocesarrego** [comment]
Hey @rijema, after check the last results from CI, I believe we have one last challenge regarding this pull, fix the Poshi tests. Can you please talk to Lino and ask his help? If you guys have difficulty to fix the tests, please reach out me. 😉 

Thanks.

**Pull request closed without merging**